api/serializers/company_serializers.py

```diff
@@ -19,5 +19,5 @@ class CompanyDetailSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Company
-        fields = ['id', 'created', 'name', 'location', 'slogan', 'logo_url', 'jobs', 'positions']
+        fields = ['id', 'created', 'name', 'location', 'slogan', 'logo_url', 'email', 'jobs', 'positions']
```

Comment:
There will be a merge conflict here when https://github.com/countryrudd/api.devlink.tech/pull/20  is merged, just giving a warning.